src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
+                {
+                    return false;
+                }
+                return _proxyPolicy != CurlProxyUsePolicy.DoNotUseProxy;
```

Comment:
> It should be FALSE only when your policy enum is DoNotUseProxy.
Ack. It is:
 
      return (_proxyPolicy != CurlProxyUsePolicy.DoNotUseProxy);